### DISCUSSION TITLE
Snap installs fail to export

### DISCUSSION THREAD

**Discussion by CappuccinoCreature** [retorquere/zotero-better-bibtex#3054] [Q&A] [answered]
When I export my Zotero library (as a Better CSL JSON file), there is no opportunity to set a file path for the exported file. I have also not been able to figure out where the exports are being stored.
I left click on 'My Library' and select 'Export Library'. I select the format 'Better CSL JSON' and check 'Keep updated'. Then, I click 'OK'. At that point, I expected a window to open that would allow me to designate a directory for the export, but nothing happens. I have tried going to Settings-->Advanced-->Data Directory Location, but the library does not appear in the directory specified there. Everything else I've read seems to indicate that this should be simple; what am I missing here?

**Comment by retorquere** [comment]
That sounds like a bug, please open an issue.

**Reply by CappuccinoCreature** [reply]
Okay, thanks. Here's a link to the issue: [Failure to export library](https://github.com/retorquere/zotero-better-bibtex/issues/3055#issue-2663285011)

**Reply by retorquere** [reply, marked as answer]
Turns out it was a Snap-install on Linux. The Zotero developers don't support snap installs, I can't either, sorry.
